src/briefcase/integrations/android_sdk.py

```diff
@@ -496,20 +496,23 @@ def select_target_device(self, device_or_avd):
         choices.append((None, "Create a new Android emulator"))
 
         # Show the choices to the user.
-        print()
-        print("Select device:")
-        print()
         try:
-            choice = select_option(choices, input=self.command.input)
+            choice = self.command.input.selection_input(
+                intro="Select device:",
+                prompt="> ",
+                options=choices,
+            )
         except InputDisabled:
             # If input is disabled, and there's only one actual simulator,
             # select it. If there are no simulators, select "Create simulator"
             if len(choices) <= 2:
                 choice = choices[0][0]
             else:
-                raise BriefcaseCommandError(
-                    "Input has been disabled; can't select a device to target."
-                )
+                raise BriefcaseCommandError("""
+Input has been disabled and the device to target is ambiguous.
```

Comment:
It took me a couple of reads to determine what this error was trying to say - my initial read is that it was logically inconsistent - either input has been disabled, *or* the device is ambiguous. The old message was a lot more direct; although I'll admit the "use -d" change is a useful addition.

Adding the -d

Reply:
yeah....i mostly just wanted to add the `use -d` because it was confusing why it failed with `--no-input`.